doc/ledger3.texi

```diff
@@ -6629,10 +6667,8 @@ to show unrealized capital in a gain/losses report.
 
 @item --rich-data
 @itemx --detail
-@c When generating ledger transaction from csv using the convert command
-@c add CSV, Imported, and UUID meta-data.
-@c see test/baeline/opt-rich-data.test
-@value{FIXME:UNDOCUMENTED}
+When generating ledger transaction from CSV file using the
+@command{convert} command, add CSV, Imported, and UUID meta-data.
```

Comment:
Please use `metadata` spelling instead of `meta-data` it is used more often and I'd like to make use of the word consistent.